test/common/upstream/upstream_impl_test.cc

```diff
@@ -900,16 +900,16 @@ TEST(HostImplTest, Weight) {
 
   EXPECT_EQ(1U, makeTestHost(cluster.info_, "tcp://10.0.0.1:1234", 0)->weight());
   EXPECT_EQ(128U, makeTestHost(cluster.info_, "tcp://10.0.0.1:1234", 128)->weight());
-  EXPECT_EQ(128U, makeTestHost(cluster.info_, "tcp://10.0.0.1:1234", 129)->weight());
+  EXPECT_EQ(10000U, makeTestHost(cluster.info_, "tcp://10.0.0.1:1234", 10000)->weight());
```

Comment:
For this and below, I think you should probably set it to the max uint32 value instead of 10000. (The fuzzers will generate arbitrary numbers in the range, so you want to guarantee in the unit test that the fuzzers won't fail later.)